src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqModelRepository.kt

```diff
@@ -1,28 +1,64 @@
 package org.vaccineimpact.api.app.repositories.jooq
 
-import org.jooq.Configuration
-import org.jooq.DSLContext
+import org.jooq.*
 import org.vaccineimpact.api.app.errors.UnknownObjectError
 import org.vaccineimpact.api.app.repositories.ModelRepository
 import org.vaccineimpact.api.db.JooqContext
-import org.vaccineimpact.api.db.Tables.MODEL
+import org.vaccineimpact.api.db.Tables.*
+import org.vaccineimpact.api.db.fromJoinPath
 import org.vaccineimpact.api.models.Model
+import org.vaccineimpact.api.models.ModelVersion
 
 class JooqModelRepository(dsl: DSLContext) : JooqRepository(dsl), ModelRepository
 {
+
+    private fun modelQuery(): SelectJoinStep<*>
+    {
+        return dsl.select(
+                MODEL.ID,
+                MODEL.DESCRIPTION,
+                MODEL.CITATION,
+                MODEL.MODELLING_GROUP,
+                MODEL.GENDER_SPECIFIC,
+                GENDER.CODE.`as`("gender"),
+                MODEL.CURRENT_VERSION)
+                .fromJoinPath(MODEL, GENDER, joinType = JoinType.LEFT_OUTER_JOIN)
+    }
+
     override fun all(): List<Model>
     {
-        return dsl.fetch(MODEL).into(Model::class.java)
+        val modelRecords =  modelQuery()
```

Comment:
Actually, should this just be getting current models? this would be a change to the current behaviour, which is to get all, but do you only want to see current in the Admin screen? Make it a qs param, 'currentOnly'?

Reply:
Should be getting current only and I don't think any need to use a query params (I can't think of a use case where we'd want to see old models, can always add that behaviour in later if we ever need it).

Reply:
Maybe an admin might? But will change as you suggest!

Reply:
I think the old ones are just there for data integrity (i.e. old estimates may have been uploaded against them) - would maybe be used in a report some time if looking at how estimates changed from prev touchstones, but not really what admins are after.